src/vector/README.md

```diff
@@ -0,0 +1,42 @@
+1. [Grafana Alloy](https://grafana.com/docs/alloy/latest/): Alloy is a distribution of the OpenTelemetry Collector, opensource and maintained by Grafana Labs. It supports the necessary [inputs and outputs](https://grafana.com/docs/alloy/latest/reference/components/) (local file/k8s logs, Loki and S3). As a distribution of OTel it supports vendor-agnostic output formats and can be integrated with numerous other tools through the OTel ecosystem. While Alloy itself is relatively new, it is built on the previous codebase of Grafana Agent and the existing OTel framework. Notably it does not have any direct integrations with Splunk or Elastic, and its S3 integration is noted as experimental.
+
+### Decision and Impact
+
+Vector has been chosen as our replacement for Promtail. Primary motivations include:
+1. Vector has an extensive "component" catalog for inputs and outputs, with complete coverage of all currently desired export locations (and all are noted as "stable" integrations).
+1. Vector's configuration is simple and works well in helm/with UDS helm overrides (easy to add additional export locations via bundle overrides for example).
+1. Despite being a newer project, Vector's community is very active - with the most active contributors and GitHub stars compared to the other two tools.
+1. Vector is [significantly more performant](https://github.com/vectordotdev/vector?tab=readme-ov-file#performance) than other tooling in the space on most categories of metrics.
+
+As with any decisions of tooling in core this can always be reevaluated in the future as different tools or factors affect how we look at our logging stack.
+
+### Upgrade Considerations
+
+During the upgrade there may be some duplication/overlap of log lines shipped to Loki due to the transition from Promtail's "position" file to Vector's "checkpoint" file (both used for tracking the last log line scraped/shipped). Grafana provides a built in feature to de-duplicate log entries when querying Loki, but this does not consistently work with all log lines due to the approach used by Grafana for de-duplication.
+
+To ensure easy querying of logs across the upgrade, all logs shipped by Vector also have a `collector` label (with the value of `vector`). This can be used to filter down any logs to those collected by either Vector or Promtail (using the `=` and `!=` operators). In general you can use these filters along with tracking your upgrade timing to properly ignore duplicate logs for the short upgrade period.
```

Comment:
Open to other thoughts on this - wanted to specifically note it as a known "limitation" here during the upgrade. Also unsure if we should relocate this doc/this portion of the doc to the `docs/` folder and docs site - or if linking this in the release notes would be sufficient?

Reply:
Also unsure if bringing this into the docs IF its a long lived doc... I think if we bring it into the docs we should add some upgrade steps / troubleshooting / expectations to the docs for the upgrade.